src/app/components/fees-row/components/custom-fee-field.tsx

```diff
@@ -59,17 +58,18 @@ export function CustomFeeField({
           height="32px"
           name="fee"
           disabled={disableFeeSelection}
-          onChange={(evt: FormEvent<HTMLInputElement>) => {
-            helpers.setValue(evt.currentTarget.value);
+          onChange={async (evt: FormEvent<HTMLInputElement>) => {
+            await helpers.setValue(evt.currentTarget.value);
             // Separating warning check from field validations
             // bc we want the user to be able to submit the form
             // with the low fee warning present.
             checkFieldWarning(evt.currentTarget.value);
           }}
           pr="38px"
           placeholder="0"
+          ring="none"
           textAlign="right"
-          type="number"
```

Comment:
Thanks @fbwoolf . I'm hopeful this will help solve this issue also #4462 . It was using a number input for some reason there which was offset in a few places.